dashboard/config/application.rb

```diff
@@ -133,5 +133,9 @@ class Application < Rails::Application
       require 'newrelic_rpm'
       require 'newrelic_ignore_downlevel_browsers'
     end
+
+    # Opt-in to new Rails 4.2 error handling in transaction callbacks.
```

Comment:
Consider removing this opt-in comment, as it will quickly become stale, after the below becomes the default.

Consider adding a TODO to remove this, after it becomes the default.